.github/workflows/main.yml

```diff
@@ -44,9 +44,9 @@ jobs:
           mv coverage/clover.xml coverage/clover-${{matrix.shard}}.xml
           mv coverage/lcov.info coverage/lcov-${{matrix.shard}}.info
           mv coverage/coverage-final.json coverage/coverage-${{matrix.shard}}.json
-      - uses: actions/upload-artifact@v3
+      - uses: actions/upload-artifact@v4
         with:
-          name: coverage-artifacts
+          name: coverage-artifacts-${{ matrix.os }}-${{ strategy.job-index }}
```

Comment:
The job is resolving this to: `coverage-artifacts--1`

@prashantasdeveloper is `.os` the right property? Shouldn't it be `matrix.shard` ?

Reply:
PR: https://github.com/PolymeshAssociation/polymesh-sdk/pull/1294